File: examples/batchMode/helloFrame.c
#include <stdio.h>
#include <stdlib.h>
  
  int main(int argc, char *argv[])
  {
    int frameNum;
    frameNum = 0;
    if (argc >= 2) {
      frameNum = atoi(argv[1]);
    }
    printf("Hello, world! From frameNum = %d\n", frameNum);
    return 0;
  }

File: examples/batchMode/helloFrame.java
package com.example;
 
public class helloFrame {
    public static void main(String[] args) {
        int frameNum = 0;
        if (args.length > 0)
            frameNum = Integer.parseInt(args[0]);
        System.out.println("Hello, world! From frameNum = " + frameNum);
    }
};
